uro98/assignment1/KToLast.java

```diff
@@ -0,0 +1,58 @@
+public class KToLast {
+
+	public static Object kToLastObject(SinglyLinkedList list, int k) {
```

Comment:
It would be much better if this method was part of the `SinglyLinkedList` rather than separate static one.